hedera-node/src/main/java/com/hedera/services/store/schedule/HederaScheduleStore.java

```diff
@@ -74,9 +74,9 @@ public HederaScheduleStore(
 			Supplier<FCMap<MerkleEntityId, MerkleSchedule>> schedules
 	) {
 		super(ids);
-		this.properties = properties;
 		this.schedules = schedules;
-		buildTxToEntityIdMap(this.schedules);
+		this.properties = properties;
+		buildContentAddressableViewOfExtantSchedules();
```

Comment:
Because we changed the order here, I think it's better to reflect that in a unit test that uses `org.mockito.InOrder`

Reply:
Not sure I understand...you mean the order of assignment for the `properties` and `schedules` fields changed?

Reply:
Ah my bad. Somehow I interpreted the assignments as some initialization methods, that was why I thought the order would matter.

Reply:
Np, thanks for the review! 👍